packages/fast-jss-manager-angular/build/tslint-watch.js

```diff
@@ -0,0 +1,27 @@
+const watch = require("watch");
```

Comment:
I guess i'm confused why in some places we're using .js and in others we're using .ts.  I believe we should be using typescript everywhere for all new stuff.

Reply:
In this case it is because I would have to install another dependency to be able to run this as a node script which seems unnecessary

Reply:
Please create a new issue to globally scrub and replace all *.js files and convert to TypeScript.